nailgun/entities.py

```diff
@@ -1670,12 +1670,21 @@ def create_missing(self):
                 'Found instance attributes: {0}'.format(self.get_values())
             )
         super(Host, self).create_missing()
+        self.name = self.name.lower()
```

Comment:
Does this indicate that host names should not include upper case characters or that the `hosts.search()` function is buggy?